docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-3.md

```diff
@@ -117,14 +117,12 @@ the following command checks that.
 test $(kubestellar-list-syncing-objects --api-group apps --api-kind ReplicaSet | grep "^ *kcp.io/cluster: [0-9a-z]*$" | sort | uniq | wc -l) -ge 2
 ```
 
-The various APIBinding and CustomResourceDefinition objects involved
+The CustomResourceDefinition objects involved
 should also appear in the mailbox workspaces.
 
 ```shell
 # TODO: kubestellar-list-syncing-objects has kcp dependencies. Will remove when controllers support spaces.
-test $(kubestellar-list-syncing-objects --api-group apis.kcp.io --api-version v1alpha1 --api-kind APIBinding | grep -cw "name: bind-apps") -ge 2
-kubestellar-list-syncing-objects --api-group apis.kcp.io --api-version v1alpha1 --api-kind APIBinding | grep -w "name: bind-kubernetes"
-kubestellar-list-syncing-objects --api-group apiextensions.k8s.io --api-kind CustomResourceDefinition | fgrep -w "name: crontabs.stable.example.com"
+kubestellar-list-syncing-objects --api-group apiextensions.k8s.io --api-kind CustomResourceDefinition | egrep -w "name: crontabs.stable.example.com|name: replicasets.apps|name: deployments.apps|name: apiservices.apiregistration.k8s.io"
```

Comment:
This test is an OR, not an AND.

Reply:
Currently (I mean what is merged now, before the effects of this PR) there are three commands. I do not see a problem with keeping multiple commands.

The effect of this PR is replacing a few APIBinding objects with a greater number of CRD objects. It would be too tedious for an example to check for all them.  I think it would be reasonable to check for just the two most prominently used in this example, the two for ReplicaSet and Deployment.

Reply:
Done.